map/src/util/Utils.js

```diff
@@ -1,5 +1,35 @@
 import TracksManager from '../context/TracksManager';
 import { apiGet } from '../util/HttpApi';
+import { useState } from 'react';
+
+/**
+ * Mutation-compatible wrapper over useState(object)
+ * Create object copy -> Apply callback() -> Call setter()
+ *
+ * Example:
+ *
+ * const [showPoints, mutateShowPoints] = useMutator({ points: true, wpts: true });
+ *
+ * mutateShowPoints((next) => (next.points = !next.points)) // functional way
+ * mutateShowPoints({ points: true, wpts: true }) // classic way supported
+ */
+export function useMutator(init) {
```

Comment:
Should to create a folder with name hooks

Reply:
Agreed. Will be done in Ksenia's branch.